bp_fe/src/include/bp_fe_decompress.vh

```diff
@@ -0,0 +1,247 @@
+`ifndef BP_FE_DECOMPRESS_VH
+`define BP_FE_DECOMPRESS_VH
+
+`ifndef BSG_DEFINES_V
+`define BSG_DEFINES_V
+`include "bsg_defines.v"
```

Comment:
These include guards are not needed.  Only have include guards in the header files themselves